Dockerfile

```diff
@@ -1,5 +1,9 @@
 # Build the manager binary
-FROM golang:1.20 as builder
+FROM registry.access.redhat.com/ubi8 as builder
```

Comment:
Can `registry.access.redhat.com/ubi8/ubi-minimal:latest` be used instead?

Reply:
@leochr I've pushed some updates to use ubi-minimal.
I've tested locally and have a pipeline build running now.